src/vss_tools/vspec/vssexporters/vss2samm/vss2samm.py

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+#
+# Convert all vspec input files to a ESMF - Aspect Model Editor (SAMM) - ttl formatted file(s).
+#
+
+
+import importlib
+import logging
+from pathlib import Path
+
+import rich_click as click
+import vss_tools.vspec.cli_options as clo
+from vss_tools import log
+from vss_tools.vspec.main import get_trees
+from vss_tools.vspec.tree import VSSNode
+
+from .config import config as cfg
+
+VSSConcepts = None
+vss_helper = None
+ttl_helper = None
+
+
+def __setup_environment(output_namespace, vspec_version, split_depth: int) -> None:
+    # Initialize config before to load other helpers / libraries, based on defined by user input, config data
+    cfg.init(output_namespace, vspec_version, split_depth)
+
+    global VSSConcepts
+    VSSConcepts = importlib.import_module("vss_tools.vspec.vssexporters.vss2samm.helpers.samm_concepts").VSSConcepts
+
+    global vss_helper
+    vss_helper = importlib.import_module("vss_tools.vspec.vssexporters.vss2samm.helpers.vss_helper")
+
+    global ttl_helper
+    ttl_helper = importlib.import_module("vss_tools.vspec.vssexporters.vss2samm.helpers.ttl_helper")
+
+
+# TODO: Currently this is a workaround to read the Vehicle.VersionVSS, which is provided from COVESA/VSS
+#       and provides the supported VSS version.
+#       In best case this functionality should be provided by the vspec tool, loaded in this script.
+#
+#       Once we migrate the vss2samm tool under the COVESA/vss-tools,
+#       we could move this functionality under the vspec script
+#       and make sure that we read the version from the COVESA/vehicle_signal_specification/VERSION
+#       or as helper function under the VSSNode - root tree.
+def __get_version_vss(vss_tree: VSSNode) -> str:
+    # DEFAULT version would be 1.0.0
+    major = 1
+    minor = 0
+    patch = 0
+
+    if vss_tree.children and len(vss_tree.children) > 0:
+        # Get the VersionVSS node so to extract current VSS version from it.
+        vss_version_node = next(filter(lambda vs_child: vs_child.name == "VersionVSS", vss_tree.children), None)
```

Comment:
There is also a tree method which might be what you want:

`vss_tree.get_child(self.get_fqn() + ".VersionVSS")`

Reply:
Done.